boards/mediatek/mt8186_adsp/mt8186_adsp.dts

```diff
@@ -0,0 +1,82 @@
+/* Copyright 2024 The ChromiumOS Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <mem.h>
+
+/dts-v1/;
+/ {
+
+#address-cells = <1>;
```

Comment:
missing indent?

Reply:
Intentionally, as the only content in the file is in the outer ("/") scope, so it would have to indent the entire file.  Same reason you don't indent for namespace declarations in C++, basically.